### PR TITLE
Add Formidable.com Header to top of OSS Landing Pages

### PR DESCRIPTION
**Pull request by knitcodemonkey**
- Add header to top of OSS pages
- Convert OSS header to new layout

This is not in it's ideal state where both the Formidable.com and the Open Source landing pages will share the same header, but it is to a point where we can update the OSS pages with the header. While the two headers are consolidated, any header updates made to Formidable.com will also need to be made here. This is meant to be a temporary situation while a solution can be found for the gatsby router being used on Formidable.com while not being used in the formidable-landers or the other OSS landing pages.

### CODE REVIEW

File: src/styles/footer.css [hunk 1]
```diff
@@ -18,12 +18,11 @@
 * .default:last-child   wrapper for trademark prop
 */
 .formidableFooter {
-  margin: 2rem 0 0 0;
+  padding: 2rem 0 0 0;
```
Comment (JonathanCooperD) [inline src/styles/footer.css]: Really trivial, but can you remove the last 0? `padding: 2rem 0 0;`

File: src/styles/base/base.css [hunk 2]
```diff
@@ -29,10 +30,21 @@
...
   flex: none;  /* Sticky header/footer setup */
-  flex-direction: row;
+  flex-direction: column;
   flex-wrap: wrap;
   justify-content: center;
-  padding: 60px 0;
+}
+.formidableHeader-container {
+  /* height: 275px; */
+  margin: 3em auto 0 auto;
```
Comment (JonathanCooperD) [inline src/styles/base/base.css]: Really trivial, but can you remove the last auto? `margin: 3em auto 0;` and for the line below.

File: src/components/header.js [hunk 3]
```diff
@@ -6,7 +6,46 @@ import LOGO_GITHUB from "../assets/logo-github.svg";
...
+      navOpen
+    });
+    this.toggleBodyClasses(navOpen);
+  };
+  /*eslint-enable no-invalid-this */
+
+  toggleBodyClasses(navOpen) {
+    const isOpenClass = "js-menu--is-open";
+
+    if (navOpen) {
```
Comment (JonathanCooperD) [inline src/components/header.js]: Could this be simplified using https://github.com/JedWatson/classnames?

File: src/styles/footer.css (hunk 1 above)
Comment (knitcodemonkey) [reply src/styles/footer.css]: Done

File: src/styles/base/base.css (hunk 2 above)
Comment (knitcodemonkey) [reply src/styles/base/base.css]: Done

File: src/components/header.js (hunk 3 above)
Comment (knitcodemonkey) [reply src/components/header.js]: I reduced this down to a ternary in the render, and got rid of this entire function.

**Comment by beccalee123** [comment]
Could you please update so the hover color on the Formidable header is the same light blue we're using on the main site? Otherwise, looks great to me! Thank you!

**Comment by knitcodemonkey** [comment]
I've updated the color to the matching light blue in the main header. All the other colors seem to match the light or dark red used in the footer, the mobile sized header, and the other links on the open source landing pages.

**Comment by beccalee123** [comment]
LGTM!

**Pull request merged**